### ISSUE TITLE
Warning of wrong data type in matplotlib's subplot

### ISSUE THREAD

**Issue by listout** [microsoft/pyright#1459]
I'm not quite sure if this is Pyright's issue or [data-science-types](https://github.com/predictive-analytics-lab/data-science-types)'s issue. This warning does not show when this **types** is not installed.

**Describe the bug**
Warning of wrong data type in matplotlib's subplot

**To Reproduce**
Minimal code given below

**Expected behavior**
No error or warnings

**Screenshots and Code**
```python
import matplotlib.pyplot as plt
import numpy as np
x = np.linspace(0, 2 * np.pi, 400)
y = np.sin(x ** 2)
fig, ax = plt.subplots()
ax.plot(x, y)
ax.set_title('A Single Plot')
plt.show()
```

![test](https://user-images.githubusercontent.com/66715002/107149412-7387af80-697e-11eb-9684-8186c06446e0.png)

**Additional context**
I'm using Neovim with coc.nvim as lsp client and coc-pyright.

**Comment by erictraut** [comment]
Pyright is doing the right thing here. The type `float64` is not compatible with the type `Data`. Please report the problem to the maintainers of the `data-science-types` stubs.
